packages/nuxt/src/utils.ts

```diff
@@ -8,3 +8,5 @@ export function toArray<T> (value: T | T[]): T[] {
 export async function isDirectory (path: string) {
   return (await fsp.lstat(path)).isDirectory()
 }
+
+export const NUXT_PREFIX = 'nuxt:'
```

Comment:
for better naming consistency, do you think we should use it everywhere (plugins + modules) to define names ?
`name: NUXT_PREFIX + 'page'`

Reply:
I think just using the string should be ok